### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by Cherkir**
changes alien speed, speeding up most aliens, slowing down corroders and praetorians(untested), changing the default mode to colonial marines, makes med vendors work, prevents hugger stacking in lockers

### CODE REVIEW

**Comment by Cherkir** [comment]
oh fuck merge conflicts AH

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -223,6 +223,8 @@
 	For most objects, pull
 */
 /mob/proc/CtrlClickOn(atom/A)
+	if (istype(A, /obj/item/clothing/mask/facehugger))
```
Comment (Carbonhell) [inline code/_onclick/click.dm]: Snowflake

Comment (Cherkir) [inline code/_onclick/click.dm]: uwot

**Comment by Carbonhell** [comment]
conflicting+already fixed stuff inside

**Pull request closed without merging**